ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -348,6 +383,7 @@ spec:
       runAfter:
         - publish-resources
         - get-supported-versions
+        - trigger-marketplace-replication
```

Comment:
I think you can remove `get-supported-version` since `trigger-marketplace-replication` depends on that. The `runAfter` clauses dictate what the pipeline diagram looks like so it'd be nice to fix this.

Reply:
Removed. The pipeline graph looks simpler now :laughing: